### DISCUSSION TITLE
ESC KEY CONFLICT

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#763] [Ideas]
### What happened?

I am developing my app and I am using keyevents to manage the navigation as it's for TV. but i have noticed that when zen is in full screen mode and when i press ESC on the keyboard then the full screen mode gets off and my navigation is also working as expected.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.29 (64-bit)

### Severity impact

Enhancement

### What platform are you seeing the problem on?

macOS - aarch64

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/688_

**Comment by irtaza9** [comment]
anything required from my side?

**Comment by stuartgelin** [comment]
Hi there, I'm running into this same issue. Is a fix for this on the roadmap? Or is there an extension that can override this?
